internal/dcache/rpc/client/functions.go

```diff
@@ -359,6 +359,96 @@ func PutChunk(ctx context.Context, targetNodeID string, req *models.PutChunkRequ
 		targetNodeID, reqStr)
 }
 
+func PutChunkEx(ctx context.Context, targetNodeID string, req *models.PutChunkExRequest) (*models.PutChunkExResponse, error) {
```

Comment:
Now that we don't have WriteMVEx() we can call this with the more appropriate name PutChunkDC() (DC for daisy chain).

Reply:
Done, rename to PutChunkDC